app/views/curation_concerns/file_sets/_show_actions.html.erb

```diff
@@ -1,5 +1,5 @@
   <div class="form-actions">
-    <%= link_to "Download this File", main_app.download_path(presenter), class: 'btn btn-default' %>
+    <%= link_to "Download this File", main_app.download_path(presenter), class: 'btn btn-default', target: :_blank %>
```

Comment:
Maybe it should be part of a separate PR, but that's the workaround for the issue in #1050.